tests/test_merger.py

```diff
@@ -96,8 +99,8 @@ def merger_operate(merger):
     found_oi = merger.find_outline_item("foo")
     assert found_oi == [9]
 
-    merger.add_metadata({"author": "Martin Thoma"})
-    merger.add_named_destination("title", 0)
+    merger.add_metadata({"/Author": "Martin Thoma"})
+    merger.add_named_destination("/Title", 0)
```

Comment:
This is a change that is not related to cloning, right? It was just wrong in the tests before.

As this PR is rather big and might take a while to review, I would like to move this to a separate PR ("Miscellaneous fixes" or similar)

Reply:
isolated into #1411